examples/deployment/assets/3-gold-asc.teal

```diff
@@ -0,0 +1,12 @@
+// Check if Sender is owner of Gold, Transaction type is pay and amount is <= 100
```

Comment:
let's have a flow in the comment same as in the code: transaction type, sender ,amount

Reply:
ok